Exercise2(rockpaperscissor))/index.js

```diff
@@ -0,0 +1,71 @@
+let counterPl = 0;
+let counterDraw = 0;
+let counterOpp = 0;
+
+const changeColors = document.getElementById("changeColors");
+changeColors.addEventListener("change", bgChange);
+
+function bgChange() {
+  document.body.style.backgroundColor = changeColors.value;
+}
+
+document.getElementById("rock").addEventListener(
+  "click",
+  (callFunction = () => {
+    getInput("rock");
+  })
```

Comment:
Que confusão, podes muito bem simplificar o código assim:
```suggestion
  () => {
    getInput("rock"); 
  }
```

Ou até mesmo assim:

```suggestion
  () => getInput("rock")
```